sdk/iot/Azure.IoT.Hub.Service/api/Azure.Iot.Hub.Service.netstandard2.0.cs

```diff
@@ -1,4 +1,4 @@
-namespace Azure.Iot.Hub.Service
+namespace Azure.IoT.Hub.Service
```

Comment:
This file should not be modified in our PR

Reply:
this file was regenerated once that I ran the command \eng\scripts\Export-API.ps1 whicha was needed becuse the build Analyse use to fail asking to re ran that command

Reply:
when you run Export-API you should pass `communication` as an argument, then it will only do it for the communication service. Now that this change is part of your branch, you either need to remove it by doing an interactive rebase or other ways to make sure your branch does not change this file in any of the commits. Or you have to abandon this branch and create a fresh branch and move your code there